src/test/scala/com.gu.bonobo/services/interpreters.scala

```diff
@@ -36,6 +46,8 @@ class BonoboServiceInterpreter extends BonoboService[TestProgram] {
     s <- State.get
     _ <- State.set((s._1, s._2, s._3 - key.consumerId))
   } yield ()
+
+  def deleteUserAndKey(user: User): TestProgram[Unit] = ???
```

Comment:
Does this need an implementation?

Reply:
To be honest I left it unimplemented on purpose as I think these tests aren't particularly useful but I can try implementing it